docs/development.rst

```diff
@@ -182,15 +195,15 @@ local clone of the zhmc-prometheus-exporter Git repo.
         MN=0.8
         BRANCH=stable_${MN}
 
-2.  Create a topic branch for the version that is being released:
+3.  Create a topic branch for the version that is being released:
```

Comment:
I think this things become rather repetitive. We should spend some time to "script" those parts that can be scripted.